packages/contracts/hardhat.config.ts

```diff
@@ -81,4 +81,16 @@ const config: HardhatUserConfig = {
   },
 };
 
+module.exports = {
```

Comment:
I do not know the side effects of this. Without this I was getting the below error:
<img width="1363" alt="Screenshot 2022-02-05 at 2 06 24 AM" src="https://user-images.githubusercontent.com/97116159/152602293-604ba6c0-1da7-4f57-8f2a-f366cc889a0d.png">

Any idea?

Reply:
I too am unsure. I wonder if we should start cutting down on the logic in `Passport.sol`. But it's possible that the optimizer just removes unused methods and fields from the dependencies. Let's see if this successfully deploys to the test networks.